internal/core/operations/remove_rooms/remove_rooms_definition.go

```diff
@@ -34,7 +34,7 @@ import (
 const OperationName = "remove_rooms"
 
 type RemoveRoomsDefinition struct {
-	Amount int `json:"amount"`
+	Amount int32 `json:"amount"`
```

Comment:
Question: Is the only reason for changing it to `int32` is to make the ProtoBuf conversion easier? If yes, should we do it? I think we shouldn't enforce anything in the core because of the API definition.

Reply:
ffdf53ba009fb7ff41b42989410c87307e736633